### PR TITLE
3.5 stable to master

### PR DESCRIPTION
**Pull request by mikz**
Merge #1006  to master.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Fixed
```
Comment (porueesq) [inline CHANGELOG.md]: How about _Enhancements and fixes_?

Comment (mikz) [reply CHANGELOG.md]: Enhancements would be in "Added".

We follow https://keepachangelog.com/en/1.0.0/

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
 
+- Fixed incorrect description of the `client` attribute in the Keycloak role check policy [PR #1005](https://github.com/3scale/APIcast/pull/1005), [THREESCALE_1867](https://issues.jboss.org/browse/THREESCALE-1867)
```
[the remark this answers is not in the document]
Comment (mikz) [reply CHANGELOG.md]: Unfortunately that change was already merged. Feel free to open a PR to master.

**Comment by mikz** [comment]
@porueesq I'll need your approval because we are merging changes to master that were already reviewed (#1006) and merged to the stable branch.

**Pull request merged**